src/root.component.test.tsx

```diff
@@ -45,35 +44,18 @@ jest.mock("@openmrs/esm-api", () => ({
 }));
 
 describe(`<Root />`, () => {
-  it(`renders without dying`, () => {
-    let wrapper;
-    act(() => {
-      wrapper = render(<Root />);
-    });
-  });
-
   it(`renders avatar`, async () => {
     mockGetCurrentUser.mockImplementation(() => of(mockUser));
-    let wrapper;
-    act(() => {
-      wrapper = render(<Root />);
-    });
+    let wrapper = render(<Root />);
     await wait(() => expect(wrapper.getByText("TU")).not.toBeNull());
   });
 
   it(`logs out patient`, async () => {
     mockGetCurrentUser.mockImplementation(() => of(mockUser));
-    let wrapper;
-    act(() => {
```

Comment:
React has gotten better in the last couple releases, such that `act()` is no longer needed when you're using @testing-library/react